build.sbt

```diff
@@ -12,6 +12,8 @@ organization in ThisBuild := "com.example"
 // the Scala version that will be used for cross-compiled libraries
 scalaVersion in ThisBuild := "2.11.8"
 
+EclipseKeys.projectFlavor in Global := EclipseProjectFlavor.Scala
+
```

Comment:
`ProjectFlavor` defaults to `Scala` in sbt-eclipse so we can get rid of this line here. :-)

https://github.com/typesafehub/sbteclipse/wiki/Using-sbteclipse#projectflavor